### PR TITLE
Modify package name and version

### PR DESCRIPTION
**Pull request by sw00**
To explicitly differentiate our private fork from that of the original, upstream package.

### CODE REVIEW

**Comment by tockards** [comment]
I am happy with the PR but I have a question. Would ` pylibmodbus-sarao` not be better as we use this package in rts, mkat and kat7. 
Just thinking about future.

File: setup.py [hunk 1]
```diff
@@ -5,19 +5,19 @@
...
 packages = [
     'pylibmodbus',
 ]
 
 setup(
-    name='pylibmodbus',
+    name='pylibmodbus-mkat',
     version=__version__,
-    description="Python wrapper for libmodbus",
+    description="Private fork of Python wrapper for libmodbus (for KATCP/meerKAT support).",
```
Comment (ajoubertza) [inline setup.py]: This repo isn't private, and not related to KATCP, so suggest:
"Python wrapper for libmodbus (fork for MeerKAT support).","

File: setup.py [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
     long_description=open('README.rst').read(),
-    author="Stéphane Raimbault",
-    author_email="stephane.raimbault@gmail.com",
+    author="SARAO meerKAT Software Team",
```
Comment (ajoubertza) [inline setup.py]: MeerKAT

**Comment by sw00** [comment]
@tockards good point but maybe it doesn't matter in this case. Consistency is generally more important than the actual name and we don't have a precedent for forks/wrappers naming schemes. 

The only other package I can think of in our project that is similar is `mkat-tango`. So perhaps you can make an argument for `mkat-pylibmodbus` instead of `pylibmodbus-mkat`.

In any case, this is something that's *very* stable and mostly unchanged in years so likely not worth agonising over.

**Pull request merged**